source/user-manual/capabilities/vulnerability-detection/offline_update.rst

```diff
@@ -102,10 +106,89 @@ In order to update the vulnerability feeds from an alternative repository, the c
         <update_interval>1h</update_interval>
     </provider>
 
+JSON feed
+^^^^^^^^^
+To perform an offline update of the Debian JSON feed, you must download the corresponding file:
+
++------------+--------------------------------------------------------------------------------------------+
+| OS         | Link                                                                                       |
++============+============================================================================================+
+| ALL        | `<https://security-tracker.debian.org/tracker/data/json>`_                                 |
++------------+--------------------------------------------------------------------------------------------+
+
+In order to use a local feed file, just use the ``path`` option:
+
+.. code-block:: xml
+
+    <provider name="debian">
+        <enabled>yes</enabled>
+        <path>/local_path/security_tracker_local.json</path>
+        <update_interval>1h</update_interval>
+    </provider>
+
+In order to update the vulnerability feeds from an alternative repository, the configuration is similar to the following:
+
+.. code-block:: xml
+
+    <provider name="debian">
+        <enabled>yes</enabled>
+        <url>/http://local_repo/security_tracker_local.json</url>
+        <update_interval>1h</update_interval>
+    </provider>
+
 Red Hat
 ^^^^^^^
+.. note:: 
+    For Red Hat we use two different feeds (OVAL and JSON), and for each one of them the offline update must be explicitly indicated; otherwise, the default mode will be assumed.
```

Comment:
Same as for Debian, add a simple explanation of what data is collected from each source